src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -191,8 +191,14 @@ internal static byte[] ConvertAlpnProtocolListToByteArray(List<SslApplicationPro
         }
 
         [GeneratedDllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_Tls13Supported")]
-        private static partial int Tls13SupportedImpl();
-        internal static readonly bool Tls13Supported = Tls13SupportedImpl() != 0;
+        private static partial int Tls13SupportedNative();
+        internal static readonly bool Tls13Supported = Tls13SupportedImpl();
```

Comment:
Rather than all this work, if the readonly-ness is still important, I'd suggest just moving the static field to a different type (just somewhere in System.Net.Security instead of in Common/src/Interop)

Reply:
I don't have any benchmarks. But it seems like doing something once is better than possibly insure some repeatedly. 
Of course, crypt is CPU expensive so the benefit may come more from code generation/elimination IMHO.